tools/crm_resource_runtime.c

```diff
@@ -1686,7 +1702,7 @@ cli_resource_execute_from_params(const char *rsc_name, const char *rsc_class,
                                  GHashTable *override_hash, int timeout_ms)
 {
     GHashTable *params_copy = NULL;
-    int rc = pcmk_ok;
+    int rc = pcmk_rc_ok;
```

Comment:
This would be one of the functions returning crm_exit_t. This above should be CRM_EX_OK, and it should probably be named something like exit_code.